src/LiquidityPool.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+
+import "../src/MyToken.sol";
+
+contract LiquidityPool {
+    MyToken public tokenA;
+    MyToken public tokenB;
+
+    mapping(address => uint256) public balancesA;
+    mapping(address => uint256) public balancesB;
+
+    uint256 public exchangeRate; // Storage variable for the exchange rate
+
+    constructor(MyToken _tokenA, MyToken _tokenB, uint256 initialRate) {
+        tokenA = _tokenA;
+        tokenB = _tokenB;
+        exchangeRate = initialRate; 
```

Comment:
`exchangeRate` should not be set here. it should be set after the first deposit i.e. the first time a liquidity provider deposit tokens A and B to the pool